src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -485,6 +487,70 @@ base::Expression opBuilderHelperStringConcat(const std::any& definition)
         });
 }
 
+// field: +s_fromArray/<separator>/<array_reference1>
+base::Expression opBuilderHelperStringFromArray(const std::any& definition)
```

Comment:
change order, first the separator and then the array

Reply:
done